slidez-fe-core/src/pages/participant-page/ParticipantPage.tsx

```diff
@@ -29,8 +30,21 @@ const ParticipantPage = () => {
         return 'watched ' + diffDate
     }
 
+    const [showQAModal, setShowQAModal] = useState(false)
+
+    const handleQAClose = () => {
+        setShowQAModal(false)
+    }
+    const handleQAShow = () => {
+        setShowQAModal(true)
+    }
+
     return (
         <div className='participant-page'>
+            <button type='button' onClick={() => handleQAShow()}>
+                Open Q&amp;A page
+            </button>
+            <Qa show={showQAModal} handleClose={() => handleQAClose()}></Qa>
```

Comment:
Alexandra added places to this button on poll component and empty page, please, add button there